core/core.go

```diff
@@ -229,25 +245,23 @@ func (c *Core) serviceBlocks(hashNumberList []types.HashAndNumber) {
 					log.Info("Requesting the dom to get the block if it doesnt have and try to append", "Hash", parentHeader.Hash(), "Order", parentHeaderOrder)
 					if c.sl.domClient != nil {
 						// send a signal to the required dom to fetch the block if it doesnt have it, or its not in its appendqueue
-						go c.sl.domClient.RequestDomToAppendOrFetch(context.Background(), parentHeader.Hash(), parentHeaderOrder)
+						go c.sl.domClient.RequestDomToAppendOrFetch(context.Background(), parentHeader.Hash(), parentHeader.ParentEntropy(), parentHeaderOrder)
 					}
 				}
 				// Using a empty block to append here because append only takes in a
```

Comment:
Update comment or remove

Reply:
what should i remove

Reply:
// Using a empty block to append here because append only takes in a // header and we read the block inside append again, so to save the // ram, we are using the header
I believe this comment is wrong now

Reply:
Removed the comment